projects/optic/src/commands/capture/actions/captureRequests.ts

```diff
@@ -202,8 +202,11 @@ async function runRequestsCommand(
         ...process.env,
         [proxyVar]: proxyUrl,
       },
-      detached: true,
+      detached: false,
```

Comment:
originally, we added this to address issues where optic would hang and processes wouldn't shut down cleanly, or was it being able to identify the PID to kill?. in haven't been able to concoct a scenario like that on mac or windows with this change.